models/topics.model.js

```diff
@@ -2,6 +2,9 @@ const db = require("../db/connection");
 
 function fetchTopics() {
   return db.query(`SELECT * FROM topics`).then(({ rows }) => {
```

Comment:
don't need the reject here - if there are no topics in the topic table then an empty array would be expected - not necessarily an error on the users part